tests/test_hic_qc.py

```diff
@@ -105,6 +105,27 @@ def test_greater_10k_contigs_right(self):
     def test_count_zero_dist_pairs(self):
         self.assertEqual(self.stats['zero_dist_pairs'], 38)
 
+    def test_reads_spanning_up_to_1k(self):
+        self.assertEqual(self.stats['reads_spanning_up_to_1k'], 101)
+
+    def test_reads_spanning_1k_to_10k(self):
+        self.assertEqual(self.stats['reads_spanning_1k_to_10k'], 0)
```

Comment:
It would be good if we had reads in the test BAM that resulted in non-zero counts for these. Maybe we could hand-create a few read pairs to add to abc_test.bam?